src/longbow/peek/command.py

```diff
@@ -0,0 +1,360 @@
+from ..utils.bam_utils import SegmentInfo
+from ..utils import model as LongbowModel
+from ..utils.model import LibraryModel
+
+
+logging.basicConfig(stream=sys.stderr)
+logger = logging.getLogger("peek")
+click_log.basic_config(logger)
+
+
+@click.command(name=logger.name)
+@click_log.simple_verbosity_option(logger)
+@click.option(
+    "-p",
+    "--pbi",
+    required=False,
+    type=click.Path(),
+    help="BAM .pbi index file",
+)
+@click.option(
+    "-t",
+    "--threads",
+    type=int,
+    default=mp.cpu_count() - 1,
+    show_default=True,
+    help="number of threads to use (0 for all)",
+)
+@click.option(
+    "-o",
+    "--output-model",
+    default="-",
+    type=click.Path(exists=False),
+    help="model name output  [default: stdout]",
+)
+@click.option(
+    "-c",
+    "--chunk",
+    type=str,
+    default="",
+    required=False,
+    help="Process a single chunk of data (e.g. specify '2/4' to process the second of four equally-sized "
+         "chunks across the dataset)"
+)
+@click.option(
+    "-n",
+    "--num-reads",
+    type=int,
+    default=100,
+    required=False,
+    help="Number of reads to examine when trying to guess the array model"
+)
+@click.option(
+    "-l",
+    "--min-length",
+    type=int,
+    default=0,
+    show_default=True,
+    required=False,
+    help="Minimum length of a read to process.  Reads shorter than this length will not be annotated."
+)
+@click.option(
+    "-L",
+    "--max-length",
+    type=int,
+    default=longbow.utils.constants.DEFAULT_MAX_READ_LENGTH,
+    show_default=True,
+    required=False,
+    help="Maximum length of a read to process.  Reads longer than this length will not be annotated."
+)
+@click.option(
+    "--min-rq",
+    type=float,
+    default=-2,
+    show_default=True,
+    required=False,
+    help="Minimum ccs-determined read quality for a read to be annotated.  CCS read quality range is [-1,1]."
+)
+@click.option(
+    '-f',
+    '--force',
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="Force overwrite of the output files if they exist."
+)
+@click.argument("input-bam", default="-" if not sys.stdin.isatty() else None, type=click.File("rb"))
+def main(pbi, threads, output_model, chunk, num_reads, min_length, max_length, min_rq, force, input_bam):
+    """Peeks at MAS-ISO-seq reads to guess the most appropriate pre-built array model to use."""
+
+    t_start = time.time()
+
+    logger.info("Invoked via: longbow %s", " ".join(sys.argv[1:]))
+
+    # Check to see if the output files exist:
+    bam_utils.check_for_preexisting_files(output_model, exist_ok=force)
+
+    threads = mp.cpu_count() if threads <= 0 or threads > mp.cpu_count() else threads
+    logger.info(f"Running with {threads} worker subprocess(es)")
+
+    # Make all prebuilt models
+    models = {}
+    for model_name in LibraryModel.pre_configured_models:
+        if not LibraryModel.pre_configured_models[model_name]['deprecated']:
```

Comment:
Can you either add an argument to include deprecated models or a warning here for each deprecated model it skips?